app/json/slate_configs.json

```diff
@@ -260,43 +260,6 @@
       }
     ]
   },
-  {
-    "id": "631d8077-1462-4397-ad0a-aa340c27570a",
```

Comment:
[Github search shows no client usage](https://github.com/search?q=org%3APocket%20aa340c27570a&type=code)

Reply:
also unable to find engagements for this slate on snowflake